core/modules/utils/utils.js

```diff
@@ -98,6 +98,31 @@ exports.pushTop = function(array,value) {
 };
 
 /*
+returns intersection of arrays
+*/
+exports.intersect = function() {
```

Comment:
@Jermolene, I have another filter now, that would need a function to return the [intersection](http://www.purplemath.com/modules/venndiag2.htm) of two attays.